src/librustdoc/html/render/print_item.rs

```diff
@@ -453,16 +454,17 @@ fn extra_info_tags(item: &clean::Item, parent: &clean::Item, tcx: TyCtxt<'_>) ->
 }
 
 fn item_function(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, f: &clean::Function) {
-    let vis = it.visibility.print_with_space(it.def_id, cx).to_string();
-    let constness = print_constness_with_space(&f.header.constness, it.const_stability(cx.tcx()));
-    let asyncness = f.header.asyncness.print_with_space();
-    let unsafety = f.header.unsafety.print_with_space();
-    let abi = print_abi_with_space(f.header.abi).to_string();
+    let header = it.fn_header(cx.tcx()).expect("printing a function which isn't a function");
+    let constness = print_constness_with_space(&header.constness, it.const_stability(cx.tcx()));
+    let unsafety = header.unsafety.print_with_space().to_string();
```

Comment:
```suggestion
    let unsafety = header.unsafety.print_with_space();
```

Reply:
```
error[E0599]: no method named `len` found for opaque type `impl std::fmt::Display` in the current scope
   --> src/librustdoc/html/render/print_item.rs:471:15
    |
471 |         + abi.len()
    |               ^^^ method not found in `impl std::fmt::Display`
```

:wink:

Reply:
:open_mouth: I'm so sorry. Don't know how I made this confusion... Sending a PR to fix this right away!

Reply:
No worries :)